src/Kestrel.Transport.Abstractions/Internal/TransportConnection.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal
 {
-    public abstract partial class TransportConnection : ConnectionContext
+    public partial class TransportConnection : ConnectionContext
```

Comment:
Isn't this breaking?

Reply:
It wasn't abstract before. I made it non-abstract so that it's not breaking.

Reply:
I see